specific/cmo_account/models/account.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+from openerp import fields, models, api
+
+
+class AccountInvoice(models.Model):
+    _inherit = "account.invoice"
+
+    quote_ref_id = fields.Many2one(
+        'sale.order',
+        string='Quotation Ref.',
+        readonly=True,
+    )
+    quote_ref_number = fields.Char(
+        string='Quotation Number',
+        states={'paid': [('readonly', True)]},
+    )
+    quote_ref_date = fields.Char(
+        string='Quotation Date',
+        states={'paid': [('readonly', True)]},
+    )
+    quote_ref_event_date = fields.Char(
+        string='Event Date',
+        states={'paid': [('readonly', True)]},
+    )
+    quote_ref_venue = fields.Char(
+        string='Venue',
+        states={'paid': [('readonly', True)]},
+    )
+    project_ref_number = fields.Char(
+        string='Project Number',
+        states={'paid': [('readonly', True)]},
+    )
+    project_ref_name = fields.Char(
+        string='Project Name',
+        states={'paid': [('readonly', True)]},
+    )
+    others_note = fields.Text(
+        string='Other',
+        states={'paid': [('readonly', True)]},
+    )
+    project_ref_id = fields.Many2one(
+         'project.project',
+         string='Project Ref.',
+         readonly=True,
+     )
+    project_note = fields.Char(
+        string='Description',
+        states={'paid': [('readonly', True)]},
+     )
+
+    @api.model
+    def create(self, vals):
```

Comment:
Change the way to pass this ID.
Better to use lower level method to pass the ID.